public/css/style.css

```diff
@@ -802,8 +802,8 @@ video {
   height: 9rem;
 }
 
-.h-64 {
-  height: 16rem;
+.h-60 {
+  height: 15rem;
```

Comment:
This shouldn't be affected by this PR. I guess that this file wasn't committed in a prev PR...

Reply:
I think you need to rebase on the current `main`, the previous one was missing all of the hugo changes.

Reply:
thanks @msujew that worked _(I wasn't aware that main wasn't the source of truth)_